### PR TITLE
feat: Tabbing Order E2E Tests - Home

### PR DESCRIPTION
**Pull request by Zeyber**
Closes: #4173

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts [hunk 1]
```diff
@@ -53,7 +60,7 @@ context("Tabbing order - tests don't require user to be logged in", () => {
     });
 
     it('should allow to navigate with tab key (filled out form)', () => {
-      loginTabbingOrder(config.login, true);
+      loginTabbingOrder(config.login);
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: why was it removed?

Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: Doesn't change anything now that we have the login button patched.

Comment (kacperknapik) [reply projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: if so, please remove that additional boolean parameter from `loginTabbingOrder()` function

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts [hunk 2]
```diff
@@ -41,20 +42,22 @@ context("Tabbing order - tests don't require user to be logged in", () => {
...
     });
   });
 
   describe('Login page', () => {
-    it('should allow to navigate with tab key (empty form)', () => {
+    it('should allow to navigate with tab key', () => {
       loginTabbingOrder(config.login);
     });
-
-    it('should allow to navigate with tab key (filled out form)', () => {
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: I wanted you to delete boolean from the functions parameters and not the whole tests :P

**Pull request merged**